content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,31 @@ processors:
   k8sattributes/default:
 ```
 
+## Setting a span status
+
+**Processor**:
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+
+Use the transform processor to set a span's status. The following example sets
+the span status to `Ok` when the `http.status_code` attribute is 400:
+
+```yaml
+# The description field allows to set a human-readable message for errors.
+transform:
+  error_mode: ignore
+  trace_statements:
+    - context: span
+      statements:
+        - set(status.code, STATUS_CODE_OK) where
+          attributes["http.request.status_code"] == 400
```

Comment:
```suggestion
        - set(status.code, STATUS_CODE_OK) where attributes["http.request.status_code"] == 200
```
Not super important, and it will work like this, but OTTL statements are generally written on a single line. I realize it's a bit long, but is it hard to read in the rendered version if it's like this?

Also, same as another comment, I think we should use a success code or try to explain a scenario where we want failure statuses to be set with an `Ok` span status.

Reply:
I have no concerns with a long code, will eave that to @Gaurang-Patel , @TylerHelmuth and @mx-psi , but see my comment above why 400

Reply:
Agree with a single line.  Although not a requirement of OTTL, it is idiomatic.

Reply:
I wrote it in single line only but actually it failed formating test , any way we can exclude this line from the test if we want to go with single line ?

Reply:
thanks @svrnm .